EasyPost.Tests/ServicesTests/ScanFormServiceTest.cs

```diff
@@ -44,7 +44,6 @@ public async Task TestAll()
 
             List<ScanForm> scanForms = scanFormCollection.ScanForms;
 
-            Assert.True(scanFormCollection.HasMore);
```

Comment:
I believe we still want this assertion even though it will always be present due to the type-strict nature of the lib, it will protect against a regression of someone removing it, or it getting overridden per object.

Reply:
Not sure why this assertion failed (was false) earlier but is true now...

Reply:
<img width="412" alt="image" src="https://user-images.githubusercontent.com/17054780/215593744-e7d1eff5-09be-4c36-bb7a-96be2784f37b.png">

Reply:
Per our internal discussions:

- This assertion will be removed, because we cannot guarantee the data returned by the API (when recording a cassette) will always return `true` for "has_more"
- The assertion is more so after confirming that "has_more" was properly deserialized, testing presence rather than value. This will be offloaded onto a standalone JSON de/serialization unit test.

Reply:
JSON de/serialization unit tests already exist to confirm that JSON data gets deserialized into models properly: https://github.com/EasyPost/easypost-csharp/blob/27037d5c2d0c2bfeeba457e832ca859bf2e4281d/EasyPost.Tests/UtilitiesTests/JsonSerializationTest.cs#L11